CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(ExaMiniMD LANGUAGES CXX)
+
+find_package(Kokkos 3.0 REQUIRED)
+option(USE_MPI "Build with MPI" ON)
+
+if (USE_MPI)
+  find_package(MPI REQUIRED CXX)
+else()
+  kokkos_check(DEVICES Cuda RETURN_VALUE USE_CUDA)
+  if (NOT USING_CUDA STREQUAL "FALSE")
+    if (CMAKE_CXX_COMPILER_ID MATCHES "nvcc")
+      enable_language(CUDA)
+      set(USE_CUDA True INTERNAL "")
```

Comment:
And remove this.

Reply:
b1a81d26ab09ff9f68aa29b7ffad18d9d40eee01